fastlane/lib/fastlane/actions/actions_helper.rb

```diff
@@ -74,6 +78,15 @@ def self.action_class_ref(action_name)
       return class_ref
     end
 
+    # load aliases of actions
+    def self.load_action_aliases
```

Comment:
Could we move the whole block from here to `def self.alias_actions` instead and cache it there in the `@alias_actions` variable? This way we can lazy init and can remove the code from `fastlane.rb` 👍